graylog2-server/src/main/java/org/graylog2/filters/ExtractorFilter.java

```diff
@@ -75,17 +76,22 @@ public boolean filter(Message msg) {
                 public List<Extractor> call() throws Exception {
                     LOG.debug("Re-loading extractors for input <{}> into cache.", inputId);
 
-                    Input input = inputService.find(inputId);
+                    try {
+                        Input input = inputService.find(inputId);
 
-                    List<Extractor> sorted = Lists.newArrayList(inputService.getExtractors(input));
+                        List<Extractor> sorted = Lists.newArrayList(inputService.getExtractors(input));
```

Comment:
Agreed, let's make the minimal change now and improve on the input service in master.